### ISSUE TITLE
[BUG] NULL value on \G - notation

### ISSUE THREAD

**Issue by mercurykd** [cweijan/vscode-mysql#899]
- Operating system: win11 x64
- Database type and version: 10.6.4-MariaDB-1:10.6.4+maria~bionic-log

I already wrote about this, I decided to make a separate issue for convenience
```sql
select * from (select 1 id, 'first' title) t1
join (select 1 id, 'two' title) t2\G
```
<image src="https://github.com/cweijan/vscode-database-client/assets/30900414/5c06c7b2-bac1-40bd-8d81-4882c414d22e" width="300">

**Comment by cweijan** [comment]
Hi, if there is no \G, is the data non-null?

**Comment by mercurykd** [comment]
> Hi, if there is no \G, is the data non-null?

yep

**Comment by cweijan** [comment]
Fixed in verison 6.7.0.
